README.md

```diff
@@ -55,7 +65,7 @@ Describe the importance of using object-oriented programming.
 
 Your Answer:
 ```js
-// your answer here
+Using OOP is important because it allows developers to better organize their code, keep it dry, and write code that represents real world events while managing different types of complex data.
```

Comment:
:+1:

I also recommend you include something about encapsulation, combining data and behavior together in one object.